### PR TITLE
PLAT-12152: adding parameters to update modules command

### PR DESCRIPTION
**Pull request by hugo-cb**
## Ticket

[PLAT-12152](https://crowdbotics.atlassian.net/browse/PLAT-12152)
_Related PRs:_ [CB app PR 4936](https://github.com/crowdbotics/crowdbotics-slack-app/pull/4936)

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [x] Minor changes

Did you make changes to the scaffold?

- [ ] Yes, and I have read the [scaffold checklist](https://crowdbotics.github.io/modules/scaffold-checklist.html) document and followed the instructions.
- [x] No.

Did you make changes to modules or created a new module?

- [ ] Yes, and have read the [modules checklist](https://crowdbotics.github.io/modules/modules-checklist.html) document and followed the instructions.
- [x] No.

## Changes introduced

Added new parameters in `update_crowdbotics_modules` command.

## Test and review

Described in [PR 4936](https://github.com/crowdbotics/crowdbotics-slack-app/pull/4936)

[PLAT-12152]: https://crowdbotics.atlassian.net/browse/PLAT-12152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alaisgomes — APPROVED** [review]
Approved, but marking as blocked until we can merge the back-end
Thanks!

**Review by danielsousaio — APPROVED** [review]
Looks good to me

**Review by cruz636 — APPROVED** [review]

**Pull request merged**
